### PR TITLE
add support for WaccaMyPageScraper CSV import

### PR DESCRIPTION
**Pull request by cg505**
This allows people to upload data from https://github.com/XezolesS/WaccaMyPageScraper, which is a widely used tool to save data from the WACCA webui (known as MyPage) before it goes offline at the end of September.
Before it goes offline, we prefer the current userscript system. However, if people join Kamaitachi after MyPage goes offline, that won't be available. But they may have saved their profile data using this other project, so it will be nice to support it for the future.

### CODE REVIEW

File: server/src/lib/score-import/import-types/file/wacca-mypage-scraper/types.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+export interface MyPageRecordsRawCsvRecord {
+
```
Comment (cg505) [inline server/src/lib/score-import/import-types/file/wacca-mypage-scraper/types.ts:2]: please tell me how to make this pass lint

Comment (zkldi) [reply server/src/lib/score-import/import-types/file/wacca-mypage-scraper/types.ts:2]: it's a cadence bug. ignore it with // eslint-ignore

**Comment by zkldi** [comment]
status check on this?

**Comment by cg505** [comment]
got busy but this is still on my radar. mostly just pending tests.

**Review by zkldi — APPROVED** [review]

**Comment by zkldi** [comment]
feels real weird having 0 open prs now...

**Pull request merged**